scripts-available/CDB_SyncTable.sql

```diff
@@ -81,7 +81,7 @@ $$ LANGUAGE sql VOLATILE PARALLEL UNSAFE;
      SELECT cartodb.CDB_SyncTable('test_sync_source', 'public', 'test_sync_dest', '{the_geom, the_geom_webmercator}');
 
 */
-CREATE OR REPLACE FUNCTION cartodb.CDB_SyncTable(src_table REGCLASS, dst_schema REGNAMESPACE, dst_table NAME, skip_cols NAME[] = '{}')
+CREATE OR REPLACE FUNCTION @extschema@.CDB_SyncTable(src_table REGCLASS, dst_schema REGNAMESPACE, dst_table NAME, skip_cols NAME[] = '{}')
```

Comment:
Any reason for using `schema.CDB_SyncTable` instead of just `CDB_SyncTable` and call it normally?

Reply:
I think it'll be better to leave out the schema for a *public* function: